### DISCUSSION TITLE
Problem with auto-correct?

### DISCUSSION THREAD

**Discussion by pianoman723** [tim-hub/obsidian-bible-reference#15] [General]
Posting here in case anyone has a similar problem. After installing this plugin, any time I typed double hyphen '--' in Obsidian, it changed to a dash '–', and therefore I can't get OBR to work. Dug through my other plugins, found I had one called 'Smart Typography' enabled, which did this exact thing. Switched off the 'Dashes' option, and works like a charm.  Thanks Tim!

**Comment by tim-hub** [comment]
Thanks for the feedback, i didn't even know that, the conflict with other plugins, 😂
